doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1716,9 +1716,9 @@ Tactic notations allow to customize the syntax of tactics. They have the followi
         - intro
 
       * - ``simple_intropattern``
-        - intro_pattern
-        - an intro pattern
-        - intros
+        - simple_intropattern
```

Comment:
More general comment about this table, it would be good if it used the token names on all rows (with the appropriate link). So this would become:

```suggestion
        - :token:`simple_intropattern`
```

and a line like:

```rst
 - qualified identifier
```

would become:

```rst
 - qualified identifier (:token:`qualid`)
```